pandas/code/pandas_file.py

```diff
@@ -0,0 +1,107 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+import re
+import os
+from sklearn.datasets import load_diabetes
+
+
+def read_gff(path):
+    pattern = re.compile("5S|16S|23S")
+    with open(path) as rna, open(".corrected_annotation.gff", mode="w") as result:
+        rna.readline()
+        for line in rna:
+            line = line.split("\t")
+            global_addition = line[8]
+            match = pattern.search(global_addition)
+            target_addition = global_addition[match.start(): match.end()]
+            line[8] = target_addition
+            line = "\t".join(line)
+            result.write(line + "\n")
+    frame = pd.read_table(".corrected_annotation.gff", sep="\t", header=None, names=["seqname", "source",
+                                                                                     "feature", "start", "end",
+                                                                                     "score", "strand", "frame",
+                                                                                     "RNA_type"])
+    os.remove('.corrected_annotation.gff')
+    return frame
+
+
+def read_bed6(path):
+    frame = pd.read_table(path, sep="\t", header=None, names=["chromosome", "start", "end", "name", "score",
+                                                              "strand"])
+    return frame
+
+
+train = pd.read_csv("../data/train.csv")
+
+# Build histograms for A,C,T,G¶
+
+nucleotides = ["A", "C", "T", "G"]
+fig, axes = plt.subplots(nrows=2, ncols=2, figsize=(10, 8))
+for i in range(2):
+    for j in range(2):
+        axes[i, j].hist(train[nucleotides[2 * i + j]])
+        axes[i, j].set_title(nucleotides[2 * i + j])
+        axes[i, j].set_xlabel("per position number")
+        axes[i, j].set_ylabel("frequency")
+fig.tight_layout()
+plt.show()
+
+# Subsetting train and writing to csv
+
+train_part = train[train["matches"] > np.mean(train["matches"])]
```

Comment:
Незачем подтягивать нампай
```suggestion
train_part = train[train["matches"] > train["matches"].mean()]
```